ui/app/styles/core/buttons.scss

```diff
@@ -129,6 +129,14 @@ $button-box-shadow-standard: 0 3px 1px 0 rgba($black, 0.12);
     text-shadow: 0 1px 1px rgba($black, 0.25);
   }
 
+  &.is-link {
```

Comment:
We need the button we use to look like a link because we were told it should be a button even if it doesn't look like one for accessibility reasons.